cfdb/harvest/harvester.py

```diff
@@ -0,0 +1,134 @@
+### Code extracted from https://github.com/regro/libcflib/blob/master/libcflib/harvester.py
```

Comment:
When vendoring (and/or modifying) code, we need to bring the license file too, along with the copyright. I'd prefer we import and wrap / patch as needed, if possible, though.